codesystems/CodeSystem-UKCore-FundingCategory.xml

```diff
@@ -20,6 +20,7 @@
   <copyright value="Copyright &#169; 2021+ HL7 UK Licensed under the Apache License, Version 2.0 (the &quot;License&quot;); you may not use this file except in compliance with the License. You may obtain a copy of the License at  http://www.apache.org/licenses/LICENSE-2.0 Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an &quot;AS IS&quot; BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License. HL7&#174; FHIR&#174; standard Copyright &#169; 2011+ HL7 The HL7&#174; FHIR&#174; standard is used under the FHIR license. You may obtain a copy of the FHIR license at  https://www.hl7.org/fhir/license.html." />
   <caseSensitive value="true" />
   <content value="complete" />
+  <hierarchyMeaning value="is-a" />
   <concept>
     <code value="nhs" />
     <display value="NHS" />
```

Comment:
Something I have just noticed, I'm assuming that NHS is meaning England. This codesystem is more 'NHS England' than 'UK Core'. I'll add a ticket to review this in the next ballot.

Reply:
Correct. should this be updated to NHS England, or merged as is now?

Reply:
Can this not just be "England" so it follow the other naming convention? If we change our name again it will not matter either.